minho/programmers/Level_1/1일차/문자열_내_p와_y의_개수.js

```diff
@@ -0,0 +1,20 @@
+function solution(s){
+    let count_p = 0;
+    let count_y = 0;
+    for(var i=0;i<s.length;i++){
+        if(s[i] === 'p' || s[i] === 'P') count_p++;
+        if(s[i] === 'y' || s[i] === 'Y') count_y++;
+    }
+    return (count_p == count_y) ? true : false;
+} //풀이 수정
+
+function solution(s){
+    let count_p = 0;
+    let count_y = 0;
+    for(let el of s.toLowerCase()) {
+        if (el === 'p') count_p++;
+        if (el === 'y') count_y++;
+    }
+    return (count_p == count_y) ? true : false;
```

Comment:
```suggestion
    return count_p === count_y;
```
제가 이 부분을 빼먹고 말씀을 안드린거같네요..ㅎ
count_p와 count_y가 같다면 true, 다르면 false 반환하므로 삼항연산자로 안써주고 바로 return 해주시면 됩니다!